phys2bids/bids.py

```diff
@@ -170,3 +172,64 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
     heurpath = os.path.join(fldr, f'{name}physio')
 
     return heurpath
+
+
+def participants_file(indir, outdir, yml, sub):
+    """[summary]
+
+    Args:
+        indir ([type]): [description]
+        outdir ([type]): [description]
+        yml ([type]): [description]
+        sub ([type]): [description]
+    """
+    LGR.info('Updating participants.tsv ...')
+    file_path = os.path.join(outdir, 'participants.tsv')
+    if not os.path.exists(file_path):
+        LGR.warning('phys2bids could not find participants.tsv')
+        # Read yaml info if file exists
+        if yml is not None and os.path.exists(os.path.join(indir, yml)):
+            LGR.info('Using yaml data to populate participants.tsv')
+            with open(os.path.join(indir, yml)) as f:
+                yaml_data = yaml.load(f, Loader=yaml.FullLoader)
+            p_id = sub
+            p_age = yaml_data['participant']['age']
+            p_sex = yaml_data['participant']['sex']
+            p_handedness = yaml_data['participant']['handedness']
+        else:
+            LGR.info('No yaml file was provided. Using phys2bids data to '
+                     'populate participants.tsv')
+            # Fill in with data from phys2bids
+            p_id = sub
```

Comment:
Can you quickly check whether this should be `f'sub-{sub}'` or only `sub`

Reply:
`sub` only returns the number. I guess it's got to be `f'sub-{sub}'`